src/components/cardList/CardList.jsx

```diff
@@ -46,14 +47,32 @@ class CardList extends React.Component {
     }
   }
 
-  applyFiltersToList(list) {
+  getSortFields(index) {
+    const { sortFields } = this.props;
+    return sortFields[index] ? sortFields[index].split('.') : '';
+
+  }
 
+  applyFiltersToList(list) {
+    const { sortFields, searchFilterFields } = this.props;
     let filters = this.props.filters ? [...this.props.filters] : [];
     list = applyFilters(list, filters, 'and');
 
-    if (this.props.searchFilterFields) {
-      list = matchSorter(list, this.state.searchValue, { keys: this.props.searchFilterFields });
-      list = matchSorter(list, this.state.searchIdentifierValue, { keys: this.props.searchFilterFields });
+    if (searchFilterFields) {
+      list = matchSorter(list, this.state.searchValue, { keys: searchFilterFields });
+      list = matchSorter(list, this.state.searchIdentifierValue, { keys: searchFilterFields });
+    }
+
+    if (sortFields) {
+      // Can accept up to 5 fallback sorting params
+      let sorter = R.sortWith([
```

Comment:
Any reason we don't just iterate over the sortfFields array and allow for 1 to n sorting params?  Doubtful we will have more than five, but iterating is less code and cleaner... :)

Reply:
Yes @mogoodrich, I tried Iterating at first, but the Ramda module, just kept throwing errors.I'll keep this in mind and try to refactor soonest.

Reply:
ah, okay, not a blocker @jomadoye ... why don't you just add a comment saying you tried iterating and that we should fix in the future as a todo... no need to worry about it now